operate/quickstart/run_service.py

```diff
@@ -136,28 +136,46 @@ def configure_local_config(template: ServiceTemplate) -> QuickstartConfig:
     if config.staking_vars is None:
         print_section("Please, select your staking program preference")
         ids = list(template["staking_programs"].keys())
-        for index, key in enumerate(ids):
-            metadata = staking_handler.get_staking_contract_metadata(program_id=key)
+        available_choices = {}
+        for index, program_id in enumerate(ids):
+            metadata = staking_handler.get_staking_contract_metadata(program_id=program_id)
             name = metadata["name"]
             description = metadata["description"]
+            available_slots = metadata["available_staking_slots"] 
             wrapped_description = textwrap.fill(
                 description, width=80, initial_indent="   ", subsequent_indent="   "
             )
-            print(f"{index + 1}) {name}\n{wrapped_description}\n")
-
+            if program_id == NO_STAKING_PROGRAM_ID:
+                print(f"{index + 1}) {name}\n{wrapped_description}\n")
+                available_choices[index + 1] = {
+                    "program_id": program_id,
+                    "slots": "∞",
+                    "name": name
+                }
+                continue
+            print(f"{index + 1}) {name} ( available slots : {available_slots} )\n{wrapped_description}\n")
+            if available_slots > 0:
+                available_choices[index + 1] = {
+                    "program_id": program_id,
+                    "slots": available_slots,
+                    "name": name
+                }
```

Comment:
after making it simpler in choose_staking.py, this can be reduced like this
```suggestion
            print(f"{index + 1}) {name}\t(available slots :{available_slots})\n{wrapped_description}\n")
            if available_slots:
                available_choices[index + 1] = {
                    "program_id": program_id,
                    "slots": available_slots,
                    "name": name
                }
```

Reply:
updated